src/redux/utilities.js

```diff
@@ -10,5 +10,9 @@ export default function formatContentType(contentType) {
     onlinereference: 'Online Reference'
   };
 
-  return contentTypes[contentType.toLowerCase()];
+  if (contentType) {
+    return contentTypes[contentType.toLowerCase()];
```

Comment:
To further bulletproof this, we could also check if the `contentTypes` hash includes this content type as well. This way if there is ever a new content type that we don't know about, this function won't return `undefined` and will just return the unformatted content type.

Reply:
:clap: Good idea. Added in new commit.